web/src/components/ContinuePreviousButtons.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { Button } from '@cmsgov/design-system-core';
+import NavLink from './NavLink';
+
+const ContinuePreviousButtons = ({ continueLink, previousLink }) => {
+  const buildLabel = (link) => {
+    if (link.url.startsWith('/apd/activity/')) {
+      const activityIndex = link.url.split('/')[3];
+      return `Activity ${+activityIndex + 1}: ${link.label}`;
+    }
+    return link.label;
+  };
```

Comment:
If you are on Activity 2 overview, the previous button is labeled as FFP and budget, and that is true but it's for Activity 1. Would it be less confusing if it said Activity 1 instead of FFP and budget?

Reply:
It used to say Activity X: FFP and Budget to show it was going back into another activity, I believe. 
Personally, I'm not 100% we need the labels under those buttons. Will look more at the p/c buttons later after meetings.

Reply:
@beparticular Thank you for your feedback! This is very helpful.

* **Clicking Export and Submit skips the review step.** Great catch! I thought something was missing, here.
* **Unable to scroll Sidebar/Nav independently from the right-hand page.** Yeah, this is bad. I must have lost an important CSS class in the refactor.
* **Continue/Previous buttons missing "Activity #" when on the main Activity List.** I can most definitely address this.
* **Click top-level navigation label, go to the first page in list.** I'll look into that. I don't think it would be too much to add.